marconi-chain-index/src/Marconi/ChainIndex/Indexers.hs

```diff
@@ -244,11 +249,12 @@ scriptTxWorker_
   -> Coordinator
   -> TChan (ChainSyncEvent (BlockInMode CardanoMode))
   -> FilePath
-  -> IO (IO (), MVar ScriptTx.ScriptTxIndexer)
+  -> IO (IO (), C.ChainPoint, MVar ScriptTx.ScriptTxIndexer)
```

Comment:
Why did you bin the indexer from the return type in the Utxo indexer and not here? it looks like it's unused, so you may drop it here as well?

Reply:
Hmm, what do you think about how to access the running indexer in general, would it be better to use the `onInsert` callback, or returning the indexer MVar? We haven't really decided on this and that is why the return types are a bit of a mess.

Reply:
My understanding is that we use callback for this purpose (it's how it's done in Marconi-sidechain to expose the indexers to json-rpc queries iirc). In that case, we can drop it in the return type.
I would suggest to try to remove them and if it breaks anything, revert.

Reply:
It's the tests that rely on the indexer being returned, so now actually did need to revert the changes.